Fix typos in constants.rb and terraform_plan_spec.rb

Pronoss opened the pull request:
1. Fixed "occurred" to "occurred" in `constants.rb` to correct the spelling of "occurred" in the fallback message.
2. Corrected "resouce" to "resource" in `terraform_plan_spec.rb` to fix the spelling of "resource" in the context description.

Pronoss committed:
fix typo constants.rb

Pronoss committed:
fix typo terraform_plan_spec.rb

cb-heimdall commented:
<h1>🟡 Heimdall Review Status</h1>
<table>
<tr>
<th>Requirement</th>
<th>Status</th>
<th>More Info</th>
</tr>
<tr>
<td>Reviews</td>
<td>
🟡
<code>0/1</code>
</td>
<td>
<details>
<summary>Denominator calculation</summary>
<table>
<tr>
<td>
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><code>1</code> if user is bot</td>
<td>0</td>
</tr>
<tr>
<td><code>1</code> if user is external</td>
<td>0</td>
</tr>
<tr>
<td><code>2</code> if repo is sensitive</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>.codeflow.yml</code>
</td>
<td>1</td>
</tr>
<tr>
<td>
Additional review requirements
<details>
<summary>Show calculation</summary>
<table>
<tr>
<td><strong>Max</strong></td>
<td><strong>0</strong></td>
</tr>
</table>
</details>
</td>
<td>0</td>
</tr>
<tr>
<td>
From <code>CODEOWNERS</code>
</td>
<td>0</td>
</tr>
<tr>
<td>Global minimum</td>
<td>0</td>
</tr>
<tr>
<td><strong>Max</strong></td>
<td>
<strong>
1
</strong>
</td>
</tr>
</table>
</details>
</td>
<td>1</td>
</tr>
<tr>
<td><code>1</code> if commit is unverified</td>
<td>0</td>
</tr>
<tr>
<td><strong>Sum</strong></td>
<td><strong>1</strong></td>
</tr>
</table>
</details>
</td>
</tr>
</table>
<!-- CODEOWNERS & Bar Raisers -->
<!-- Power Rangers -->
<!-- Links -->

cb-heimdall commented:
Review Error for BesBettison @ 2025-06-30 16:28:15 UTC
User must have write permissions to review

cb-heimdall commented:
Review Error for BesBettison @ 2025-07-03 12:21:51 UTC
User must have write permissions to review

Pronoss commented:
Hi team 👋 – friendly reminder about PR #151 (“Fix typos in constants.rb and terraform_plan_spec.rb”):

- **constants.rb**: corrected the spelling of “occurred” in the fallback message  
- **terraform_plan_spec.rb**: fixed “resouce” → “resource” in the context description  

Thanks for reviewing when you have a moment! 😊
